ibm_mq/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - IBM MQ
 
+## 4.0.0 / 2020-03-11
+
+* [Fixed] IBM MQ refactor. See [#5902](https://github.com/DataDog/integrations-core/pull/5902).
+* [Changed] Add configuration option for the Channel Definition API version. See [#5905](https://github.com/DataDog/integrations-core/pull/5905).
+* [Changed] Upgrade pymqi to 1.10.1. See [#5955](https://github.com/DataDog/integrations-core/pull/5955).
```

Comment:
Shouldn't these be "Added"? Don't seem to warrant a major bump.

Reply:
You're right